deepchecks/nlp/utils/text_properties.py

```diff
@@ -152,37 +198,42 @@ def formality(raw_text: Sequence[str], device: Optional[int] = None) -> List[flo
     {'name': 'Toxicity', 'method': toxicity, 'output_type': 'numeric'},
     {'name': 'Fluency', 'method': fluency, 'output_type': 'numeric'},
     {'name': 'Formality', 'method': formality, 'output_type': 'numeric'}
-]
+)
 
 LONG_RUN_PROPERTIES = ['Toxicity', 'Fluency', 'Formality']
 ENGLISH_ONLY_PROPERTIES = ['Sentiment', 'Subjectivity', 'Toxicity', 'Fluency', 'Formality']
 LARGE_SAMPLE_SIZE = 10_000
 
 
-def _get_default_properties(include_properties: List[str] = None, ignore_properties: List[str] = None):
+def _get_default_properties(
+    include_properties: Optional[List[str]] = None,
+    ignore_properties: Optional[List[str]] = None
+):
     """Return the default properties.
 
     Default properties are defined here and not outside the function so not to import all the packages
     if they are not needed.
     """
-    ret_properties = DEFAULT_PROPERTIES
+    properties = DEFAULT_PROPERTIES
 
     # Filter by properties or ignore_properties:
     if include_properties is not None and ignore_properties is not None:
         raise ValueError('Cannot use properties and ignore_properties parameters together.')
     elif include_properties is not None:
-        ret_properties = [prop for prop in ret_properties if prop['name'] in include_properties]
+        properties = [prop for prop in properties if prop['name'] in include_properties]
     elif ignore_properties is not None:
-        ret_properties = [prop for prop in ret_properties if prop['name'] not in ignore_properties]
+        properties = [prop for prop in properties if prop['name'] not in ignore_properties]
 
-    return ret_properties
+    return properties
 
 
-def calculate_default_properties(raw_text: Sequence[str], include_properties: Optional[List[str]] = None,
-                                 ignore_properties: Optional[List[str]] = None,
-                                 include_long_calculation_properties: Optional[bool] = False,
-                                 device: Optional[str] = None
-                                 ) -> Tuple[Dict[str, List[float]], Dict[str, str]]:
+def calculate_default_properties(
```

Comment:
export this function to be used directly from deepchecks.nlp.utils

Reply:
fixed